src/com/firebirdberlin/nightdream/Settings.java

```diff
@@ -351,6 +355,25 @@ public void reload() {
         }
     }
 
+    boolean getPreventBurnIn() {
+        boolean on = settings.getBoolean("preventBurnIn", true);
+
+        if (!restless_mode){
+            on = false;
+            setPreventBurnIn(on);
+            setRestlessMode(true);
+        }
```

Comment:
It is not necessary to write restlessMode. We want to get rid of this setting. Instead I would do 

```suggestion
        if (settings.contains("restlessMode"){
            Editor edit = settings.edit()
            edit.setString("screenProtectionMode", 1)
            edit.remove("restlessMode");
            edit.apply;
        }
       return settings.getString("screenProtectionMode", 1); 
```

If you do so, you are able to delete all other occurrences of restless_mode in the code.